packages/@coorpacademy-components/src/molecule/dashboard/cards-list/index.js

```diff
@@ -106,7 +107,7 @@ class CardsList extends React.Component {
   }
 
   cardsWidth() {
-    return this.cards.map(getOr(0, 'scrollWidth')).reduce((a, b) => a + b, 0);
+    return map(getOr(0, 'scrollWidth'), this.cards).reduce((a, b) => a + b, 0);
```

Comment:
Simplifiable par sumBy de Lodash
`sumBy('scrollWidth', this.cards)`

Il manque de la couverture sur cette ligne, ce serait pas mal si tu pouvais rajouter un test qui vérifie les retours de cette fonction.